shimmingtoolbox/unwrap/__init__.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "0.1"
```

Comment:
we only need to mention the version in the main `shimmingtoolbox/__init__.py`, right? otherwise there is duplication